types/incidentcreationconditions.pp

```diff
@@ -0,0 +1,26 @@
+# The possible incident creation conditions. Here's what each condition means:
+#   'always'
```

Comment:
I thought about `no_changes` for some time, and to me it felt like such a thing would only be useful if users wanted to include `no_changes` reports but _not_ the other kinds of reports (e.g. `corrective_changes`, `failures`, etc.). That felt like a theoretical possibility vs. a practical possibility.

I think the real intent with `no_changes` was `always`, but reason latter was removed was b/c it felt weird to display it as a report attribute in the previous naming. It also feels weird to display it as a `ReportCondition` (what does the `always` report condition mean?). However, changing the type to `IncidentCreationConditions`, the `always` condition makes a lot more sense so I removed `no_changes` and replaced it with `always`.